### PR TITLE
(maint) Update CHANGELOG.md for 2.0.0

### PR DESCRIPTION
**Pull request by adrienthebo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This is a major release
 
-* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) Move Jetty dependency to 9.4.1
+* [TK-369](https://tickets.puppetlabs.com/browse/TK-369) [TK-440](https://tickets.puppetlabs.com/browse/TK-440) Move Jetty dependency to 9.4.4.
```
Comment (jpinsonault) [inline CHANGELOG.md]: You have two links here

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ This is a major release
   encoded string.
   (https://github.com/eclipse/jetty.project/commit/7f62f2600b943b9aed0e4771891939bf61372c5a)
 
+* [TK-373](https://tickets.puppetlabs.com/browse/TK-373) Enable Diffie Hellman
+  cipher suites.  DHE cipher suites were that were previously disabled due to
```
Comment (rlinehan) [inline CHANGELOG.md]: I think there's an extra "were that" here.

File: CHANGELOG.md (hunk 1 above)
Comment (adrienthebo) [inline CHANGELOG.md]: That's intentional as we had multiple tickets for bumping the Jetty version. I can just pick one if you prefer.

**Comment by rlinehan** [comment]
one typo, otherwise 👍

File: CHANGELOG.md (hunk 1 above)
Comment (jpinsonault) [inline CHANGELOG.md]: Ah ok, no that's fine then

File: CHANGELOG.md (hunk 2 above)
Comment (adrienthebo) [inline CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (adrienthebo) [inline CHANGELOG.md]: I opted to drop the TK-440 reference and stick with TK-369.

**Pull request merged**